inttest/util/helm.go

```diff
@@ -0,0 +1,59 @@
+package util
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func WriteHelmValuesFile(t *testing.T, name string, values string) string {
+	f, err := os.CreateTemp("", "values-*.yaml")
+	if err != nil {
+		t.Fatalf("failed to create temp file: %s", err)
+	}
+	f.Close()
+
+	err = os.WriteFile(f.Name(), []byte(values), 0644)
```

Comment:
could use the `WriteTempFile` function you have elsewhere

Reply:
done